packages/shared/lib/participation/constants.ts

```diff
@@ -50,6 +50,8 @@ export const STAKING_AIRDROP_TOKENS: { [key in StakingAirdrop]: string } = {
  */
 export const PARTICIPATION_POLL_DURATION = 10 * MILLISECONDS_PER_SECOND
 
+export const ASSEMBLY_STAKING_BLOG_POST_URL = 'https://blog.iota.org/iota-staking-for-assembly-round-6/'
```

Comment:
Not sure whether you need this on mobile, since you don't seem to have it referenced in the code anywhere.

Reply:
Thanks you're right, I think would be good to add it on the Assembly drawer

Reply:
Yup, although, it's probably out-of-scope, since we need a release in 3 hours.